gsa/src/web/pages/tickets/dashboard/usersassigneddisplay.js

```diff
@@ -0,0 +1,84 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+import {scaleLinear} from 'd3-scale';
+
+import {_, _l} from 'gmp/locale/lang';
+
+import {TICKETS_FILTER_FILTER} from 'gmp/models/filter';
+
+import BubbleChart from 'web/components/chart/bubble';
+
+import DataDisplay from 'web/components/dashboard/display/datadisplay';
+
+import {TicketsListLoader} from './loaders';
+import {registerDisplay} from 'web/components/dashboard/registry';
+import createDisplay from 'web/components/dashboard/display/createDisplay';
+
+export const assignedUserColorScale = scaleLinear()
+  .domain([0, 0.05, 0.25, 0.50, 0.75, 0.95, 1.00])
+  .range([
+    '#008644',
+    '#55B200',
+    '#94D800',
+    '#E6E600',
+    '#EDBA00',
+    '#EC6E00',
+    '#D63900',
+  ]);
+
```

Comment:
Not sure about handling the colors. The BubbleChart is about the size of bubbles, not their color. What exactly does this do?